docs/howto/manage-charms.rst

```diff
@@ -89,14 +91,6 @@ to ``charm`` in your project file:
 Specify a name
 ~~~~~~~~~~~~~~
 
-.. note::
```

Comment:
Note that this isn't removed, just moved, as the expectation is that charmers are most likely to notice it when using the reference rather than the how-to.